### PR TITLE
feature/processYOLOv3 read output of library YOLOv3 and mark the boxes

### PR DESCRIPTION
**Pull request by LuisRBarreras**
- Process output of YOLOv3 JSON file
- Create function mark boxes, this function draw all boxes of objects detected

### CODE REVIEW

File: src/tools/loadData.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const data = require('../../data/yolo3.json');
```
Comment (keogh) [inline src/tools/loadData.js]: se puede usar `import` en lugar de `require`?

File: src/tools/loadData.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+let read = () => {
```
Comment (keogh) [inline src/tools/loadData.js]: `const` instead of `let` ya que la variable `read` no va a cambiar de valor

File: src/tools/mark.js [hunk 3]
```diff
@@ -1,8 +1,26 @@
 
-function mark() {
-  var ctx = document.getElementById('canvas').getContext('2d');
-  ctx.rect(20, 20, 400, 150);
+const colors = {
+  truck: 'blue',
+  car: 'green'
+}
+
+function mark({ x, y, width, height, label }) {
```
Comment (keogh) [inline src/tools/mark.js]: deconstructing ftw

File: src/components/Canvas.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+import draw from '../tools/draw';
+import markBoxes from '../tools/mark';
+import read from '../tools/loadData';
+
+
+const Canvas = () => {
+  const [mark, setMark] = useState(false);
+
+  useEffect(() => {
```
Comment (keogh) [inline src/components/Canvas.jsx:11]: no conozco `useEffect` 🤔

**Review by keogh — APPROVED** [review]
Great job! Solo unos commentarios no bloqueadores :)

File: src/components/Canvas.jsx (hunk 4 above)
Comment (LuisRBarreras) [reply src/components/Canvas.jsx:11]: Checar
> You can think of useEffect Hook as componentDidMount, componentDidUpdate, and componentWillUnmount combined.
[useEffect](https://reactjs.org/docs/hooks-effect.html)

File: src/tools/loadData.js (hunk 1 above)
Comment (LuisRBarreras) [reply src/tools/loadData.js]: Hecho

File: src/tools/loadData.js (hunk 2 above)
Comment (LuisRBarreras) [reply src/tools/loadData.js]: Hecho

File: src/tools/mark.js (hunk 3 above)
Comment (LuisRBarreras) [reply src/tools/mark.js]: Ok, lo remuevo

**Pull request merged**